src/main/scala/is/hail/methods/LDPrune.scala

```diff
@@ -2,15 +2,68 @@ package is.hail.methods
 
 import java.util
 
+import is.hail.HailContext
 import is.hail.annotations._
 import is.hail.expr.types._
+import is.hail.linalg.BlockMatrix
 import is.hail.sparkextras.GeneralRDD
 import org.apache.spark.storage.StorageLevel
-import is.hail.sparkextras._
 import is.hail.rvd.{OrderedRVD, OrderedRVDType, RVD, UnpartitionedRVD}
+import is.hail.table.Table
 import is.hail.variant._
 import is.hail.utils._
-import org.apache.spark.sql.Row
+
+import scala.collection.mutable
+
+object LDPruneSparseEntriesTable {
+  def windows(tbl: Table, window: Int): List[(Long, Long)] = {
+    val expectedSignature = TStruct("index" -> TInt64(), "chromosome" -> TString(), "position" -> TInt32())
+    if (tbl.signature != expectedSignature) {
+      fatal(s"Expected table to have signature $expectedSignature but found ${ tbl.signature }")
+    }
+
+    val q = new mutable.Queue[(Long, String, Int)] // index, chromosome, position
+    var stack = List[(Long, Long)]()
```

Comment:
`q` and `stack` are generic names, making it hard for me to discern their purpose in the algorithm. Can you make the names more descriptive?